docs/user_guide/hopsworks/search.rst

```diff
@@ -0,0 +1,70 @@
+Search
+========================
+
+The search on the navigation bar can be used to search for Projects, Datasets, Feature groups, Features and Training datasets.
+Depending on the page you are in, landing page or project dashboard, the search scope will cover all projects or the specific project you are in.
+
+.. figure:: ../../imgs/search/global_search.png
+    :alt: Global Search
+    :scale: 30%
+    :figclass: align-center
+
+In the case of searching on the landing page, as we can see in the image above, your search will target all items that have not been marked explicitly as being private when they were create. By default all the metadata of items(Projects, Datasets, Feature Groups, Features, Training Datasets) are public and searchable by all users in the cluster. The data however is always private and requires access granting by the owners in order to have access to it. When searching globaly you can narrow your search to the categories: Projects/Datasets/Featurestore(Featuregroup, Features and Training Datasets) by selecting the appropriate drop down option.
```

Comment:
```suggestion
In the case of searching on the landing page, as we can see in the image above, your search will target all items that have not been marked explicitly as being private when they were created. By default all the metadata of items(Projects, Datasets, Feature Groups, Features, Training Datasets) are public and searchable by all users in the cluster. The data however is always private and requires access granting by the owners in order to allow access. When searching globally you can narrow your search to the categories: Projects/Datasets/Featurestore(Featuregroup, Features and Training Datasets) by selecting the appropriate drop down option.
```

Reply:
I think allow access will change the meaning of the sentence.